package.json

```diff
@@ -19,12 +19,12 @@
     "test": "mocha --recursive tests/"
   },
   "engines": {
-    "node": ">=6.10.0"
+    "node": ">=10.0.0"
   },
   "author": "Button <info@usebutton.com>",
   "license": "MIT",
   "dependencies": {
-    "google-oauth-jwt": "^0.2.0"
+    "request": "^2.88.2"
```

Comment:
request.js is currently considered a "completed" project, and I took a quick look at alternatives before deciding it made some sense to stick with it: we return the request.js response object to callers through public APIs like `MinQuery#insert`, and it's not worth breaking them IMO.